src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -219,311 +176,37 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
-    if (isTokenDeposit) return
-    if (!contextDispatch) return
-    if (!(accountId && assetId && feeAsset)) return
-    if (!state?.deposit.cryptoAmount) {
-      throw new Error('Cannot send 0-value THORCHain savers Tx')
-    }
-
-    try {
-      const estimatedFees = await getEstimatedFees()
-      if (!estimatedFees) return
-      setNetworkFeeCryptoBaseUnit(estimatedFees.fast.txFee)
-      contextDispatch({
-        type: ThorchainSaversDepositActionType.SET_DEPOSIT,
-        payload: {
-          networkFeeCryptoBaseUnit: estimatedFees.fast.txFee,
-        },
-      })
-
-      const amountCryptoBaseUnit = toBaseUnit(state.deposit.cryptoAmount, asset.precision)
-
-      const maybeQuote = await getMaybeThorchainSaversDepositQuote({ asset, amountCryptoBaseUnit })
-      if (maybeQuote.isErr()) throw new Error(maybeQuote.unwrapErr())
-      const quote = maybeQuote.unwrap()
-
-      let maybeGasDeductedCryptoAmountCryptoPrecision = ''
-      if (isUtxoChainId(chainId)) {
-        if (!maybeFromUTXOAccountAddress) {
-          throw new Error('Account address required to deposit in THORChain savers')
-        }
-
-        // DO NOT MAKE ME A `useAppSelector()` HOOK
-        // DO NOT extract store.getState() to a component/module-scope variable
-        // React reconciliation algorithm makes it so this wouldn't change until the next time this is fired
-        // But the balance actually changes from the gas fees of the reconciliation Tx if it's fired
-        // So the next time we fire the actual send Tx, we should deduct from the udpated balance
-        const assetBalanceCryptoBaseUnit = selectPortfolioCryptoBalanceBaseUnitByFilter(
-          store.getState(),
-          assetBalanceFilter,
-        )
-        const fastFeesBaseUnit = estimatedFees.fast.txFee
-
-        setNetworkFeeCryptoBaseUnit(fastFeesBaseUnit)
-        contextDispatch({
-          type: ThorchainSaversDepositActionType.SET_DEPOSIT,
-          payload: {
-            networkFeeCryptoBaseUnit: fastFeesBaseUnit,
-          },
-        })
-
-        const cryptoAmountBaseUnit = toBaseUnit(state.deposit.cryptoAmount, asset.precision)
-
-        const needsFeeDeduction = bn(cryptoAmountBaseUnit)
-          .plus(fastFeesBaseUnit)
-          .gte(assetBalanceCryptoBaseUnit)
-
-        if (state?.deposit.sendMax) {
-          maybeGasDeductedCryptoAmountCryptoPrecision = bnOrZero(assetBalanceCryptoBaseUnit)
-            .minus(bn(fastFeesBaseUnit).times(TXS_BUFFER))
-            .div(bn(10).pow(asset.precision))
-            .toFixed()
-        } else if (needsFeeDeduction)
-          // We tend to overestimate so that SHOULD be safe but this is both
-          // a safety factor as well as ensuring we keep a bit of gas away for another Tx
-          maybeGasDeductedCryptoAmountCryptoPrecision = bnOrZero(state.deposit.cryptoAmount)
-            .minus(bn(fastFeesBaseUnit).times(TXS_BUFFER).div(bn(10).pow(asset.precision)))
-            .toFixed()
-      }
```

Comment:
more send max/fees logic that is being removed/changed. not sure the relevance yet, but making note